pkgs/development/python-modules/watermark/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, importlib-metadata
+, ipython
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "watermark";
+  version = "2.2.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "rasbt";
+    repo = pname;
+    rev = "d3553b68dd30ac5b0951a6fae6083236e4c7f3bd";
```

Comment:
Any reason why you can't use the tag you set in version?

Reply:
Inattention mostly, I will remember it next time, thank you for pointing it out.

Reply:
Please patch this to make auto updates work.